user-access-handlers/src/test/java/no/unit/nva/handlers/CreateUserHandlerTest.java

```diff
@@ -82,9 +82,12 @@ public void close() {
     @Test
     void shouldCreateUserWithRequestedRolesWhenInputContainsNationalIdNumberNvaCustomerIdAndSetOfRoles()
         throws IOException {
-        var requestBody = sampleRequestForExistingPersonCustomerAndRoles();
 
-        var request = createRequest(requestBody, EDIT_OWN_INSTITUTION_USERS);
+        NationalIdentityNumber person = peopleAndInstitutions.getPersonWithSomeActiveAndSomeInactiveAffiliations();
+        var someCustomer = fetchSomeCustomerForThePerson(person);
+        var requestBody = sampleRequestForExistingPersonCustomerAndRoles(person, someCustomer.getId());
+
+        var request = createRequest(requestBody, someCustomer, EDIT_OWN_INSTITUTION_USERS);
```

Comment:
has to repeat these lines in tests because the following test uses the CristinId